bedita-app/vendors/shells/editorial_contents.php

```diff
@@ -38,23 +38,53 @@ class EditorialContentsShell extends BeditaBaseShell
      */
     public function help() {
         $this->out('Update `publisher` field for all users that belong to a group');
-        $this->out('that grants them access to backend app.');
+        $this->out('that grants them access to backend app with default publisher configurated.');
+        $this->out();
+        $this->out('  Usage: editorial_contents [--reset-empty-publisher]');
+        $this->out();
+        $this->out('    --reset-empty-publisher change object publisher from empty string to NULL value');
         $this->out();
 
-        $this->out('  Usage: editorial_contents');
     }
 
     /**
      * Convert all date items in the database.
      */
     public function main() {
-        // Check that we actually have something to update.
-        $publisher = Configure::read('editorialContents.defaultPublisher');
-        if (empty($publisher)) {
-            $this->out('=====> Configuration item `editorialContents.defaultPublisher` must not be empty');
-            $this->err('Aborting');
 
-            exit(1);
+        $resetEmptyPublisher = !empty($this->params['-reset-empty-publisher']);
+
+        if ($resetEmptyPublisher) {
+
+            $backendAuth = 0;
```

Comment:
Is there a specific reason because the reset should work just for objects created from users belong to frontend groups?

Reply:
You are right, the reset must work for all objects independently abou user who created it